.github/workflows/ci.yml

```
@@ -208,6 +208,18 @@ jobs:
             source plugin/trino-hive-hadoop2/conf/hive-tests-${{ matrix.config }}.sh &&
               plugin/trino-hive-hadoop2/bin/run_hive_s3_tests.sh
           fi
+      - name: Run Hive S3 Select JSON Tests
+        if: matrix.config == 'config-hdp3'
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESSKEY }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRETKEY }}
+          S3_BUCKET: "presto-ci-test"
+          S3_BUCKET_ENDPOINT: "s3.us-east-2.amazonaws.com"
+        run: |
+          if [ "${AWS_ACCESS_KEY_ID}" != "" ]; then
+            source plugin/trino-hive-hadoop2/conf/hive-tests-config-hdp3.sh &&
```

How about using the matrix value here for consistency with other steps? The step condition is now sufficient to guard this, but might be removed in future.
```suggestion
            source plugin/trino-hive-hadoop2/conf/hive-tests-${{ matrix.config }}.sh &&
```

---

Yes, sounds good. Updated the script now.